.github/workflows/test.yml

```diff
@@ -0,0 +1,91 @@
+name: Tests
+on:
+  push:
+    branches:
+      - "*"
```

Comment:
Yeah that's probably better.
Alternatively, maybe I should actually look into running go drink from compose instead of github action services?

In that case, the action would also test whether the compose-files are actually working.

Or more generally speaking, moving from Github-Magic to more portable scripting.

But I think I'll postpone that to work on actual code for now and therefore I'll have a look at restraining the action trigger.

Reply:
I excluded almost the same paths as in docker.yaml (I did _not_ exclude the http-test folder for obvious reasons)

Reply:
looks good to me :) I added the Dockerfile to the list as well, since for the tests, we run the binary without anything docker...